loguru_example/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-missing-noreturn")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-missing-prototypes")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-gnu-zero-variadic-macro-arguments")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-disabled-macro-expansion")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-implicit-int-float-conversion")
```

Comment:
I think it would be better to fix [the offending line](https://github.com/emilk/loguru/blob/master/loguru.cpp#L1443) with a simple cast, e.g.

``` C++
auto duration_sec = static_cast<double>(now_ns() - _start_time_ns) / 1e9;
```